subprojects/base-services/src/test/groovy/org/gradle/internal/jvm/JvmTest.groovy

```diff
@@ -136,18 +138,23 @@ class JvmTest extends Specification {
             }
         }
 
-        expect:
-        def jvm = new Jvm(os, software.file('jdk/jre'), JavaVersion.current());
+        when:
+        def jvm = new Jvm(os, software.file('jdk'), "1.8.0.221", JavaVersion.VERSION_1_8)
```

Comment:
❌ It is not clear from the context whether the constant `"1.8.0.221"` matters and how, whether it must match or not an installed version. 
It would make sense to introduce an explaining variable or field.